runtime-sdk/src/modules/core/mod.rs

```diff
@@ -212,7 +223,25 @@ impl module::Module for Module {
     type Parameters = Parameters;
 }
 
-impl module::AuthHandler for Module {}
+impl module::AuthHandler for Module {
+    fn approve_utx(
+        ctx: &mut DispatchContext<'_>,
+        utx: &UnverifiedTransaction,
+    ) -> Result<(), Error> {
+        let params = Self::params(ctx.runtime_state());
+        if utx.1.len() > params.max_tx_signers as usize {
+            return Err(Error::TooManyAuth);
```

Comment:
Should add test cases for these two failure cases.

Reply:
added!